src/directives/model/default.js

```diff
@@ -31,7 +34,7 @@ module.exports = {
     // shared setter
     function set () {
       self.set(
-        number ? _.toNumber(el.value) : el.value,
+        file ? el.files : ( number ? _.toNumber(el.value) : el.value ),
```

Comment:
Oh, but doing that would make it un-passable to other browser APIs, thus making it useless.